src/components/MemberSearchPage/MemberSearchContainer.js

```diff
@@ -77,9 +81,13 @@ class MemberSearchContainer extends Component {
       queryString,
       rolesFilter,
       interestsFilter,
+      adminOptionsFilter,
       currentPageUpdate,
     } = this.props;
-    const maxPage = count.public / membersPerPage;
+
+    const showAll = adminOptionsFilter['allMembers'] && this.props.isAdmin;
```

Comment:
adminOptionsFilter.allMembers looks better to me. + why not include isAdmin in the deconstructing object above? {otherListedProps, currentPageUpdate, isAdmin } = this.props;